app/src/screens/MapScreen.js

```diff
@@ -74,15 +78,15 @@ function MapScreen(): React.Node {
   const [unfilteredLayers, setUnfilteredLayers] = useState<LayerDict>({});
   const [filteredLayers, setFilteredLayers] = useState<LayerDict>({});
   const [hoverInfo, setHoverInfo] = useState<?PickInfo<Point>>(null);
-  const [minRssiToDisplay, setMinRssiToDisplay] = useState<number>(-1000);
-  const [maxRssiToDisplay, setMaxRssiToDisplay] = useState<number>(0);
   const [mapStyle, setMapStyle] = useState<string>(
     'mapbox://styles/mapbox/satellite-v9',
   );
   const [satelliteView, setSatelliteView] = useState<boolean>(true);
   const [cacheMapDialogOpen, setCacheMapDialogOpen] = useState<boolean>(false);
   const [filterMinRssi, setFilterMinRssi] = useState('');
   const [filterMaxRssi, setFilterMaxRssi] = useState('');
+  const [filterMinHeight, setFilterMinHeight] = useState('10');
```

Comment:
Y U no use a number instead of a string?

Reply:
@murtadha did it first, and I followed LOL
The main reason is because the input is in a text field and thus a string regardless. Instead of trying to always parse it, this works easily....